app/javascript/hyacinth_v1/components/digital_objects/system_data/SystemData.js

```diff
@@ -29,10 +36,23 @@ function SystemData(props) {
   const { digitalObject } = digitalObjectData;
 
   const canDeleteObject = digitalObjectAbility.can('delete_objects', { primaryProject: digitalObject.primaryProject, otherProjects: digitalObject.otherProjects });
+  const canPurgeObject = digitalObjectAbility.can('purge_objects', { primaryProject: digitalObject.primaryProject, otherProjects: digitalObject.otherProjects });
+
 
   const onDelete = (e) => {
     e.preventDefault();
-    digitalObjectApi.delete(id).then(() => history.push('/digital_objects'));
+
+    const variables = { input: { id: digitalObject.id } };
+
+    deleteDigitalObject({ variables }).then(() => history.push('/digital_objects'));
```

Comment:
Since delete is a more dangerous operation, would you mind wrapping this operation in an `if(window.confirm('Are you sure you want to delete this object?'))`?

Reply:
Never mind!  I forgot that this logic exists in the DeleteButton component!